OWNERS_ALIASES

```diff
@@ -245,11 +245,12 @@ aliases:
   - childsb
   - bertinatto
   openstack-approvers:
-  - flaper87
+  - mandre
+  - Fedosin
+  - iamemilio
   - tomassedovic
   - hardys
```

Comment:
hardys? russellb? are also working on openstack-ci ?

Reply:
We do not know where this is synced from, so we are going to leave it as is and list the approvers by name for now. Once we are not Teetering on the edge of GA, we have an action item to move this to its own directory where we are the owners of it, and properly update the upstream owners file.